vision/product_search/spec/product_search_purge_products_in_product_set_spec.rb

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 Google, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "spec_helper"
+
+describe "Purge products in product set" do
+  example "Purge products in product set" do
+    snippet_filepath = get_snippet_filepath __FILE__
+    temp_product = create_temp_product
+    temp_product_id = get_id temp_product
+    temp_product_set = create_temp_product_set [temp_product]
+    temp_product_set_id = get_id temp_product_set
+    product_list_before = Array(@client.list_products_in_product_set(temp_product_set.name))
+    expect(product_list_before.length).to eq 1
+
+    output = `ruby #{snippet_filepath} #{@project_id} #{@location} #{temp_product_set_id}`
+
+    expect {
```

Comment:
```suggestion
    product_list_after = Array(@client.list_products_in_product_set(temp_product_set.name))
    expect(product_list_after.length).to eq 0
    
    # Verify product was deleted
    expect {
```

 - There is a positive assertion for checking there is 1 product in the product set.
 - Recommend adding the equivalent negative assertion after performing the action (running sample).